### PR TITLE
Sort skills list results

### PR DESCRIPTION
**Pull request by michael616kriel**
Sort skills list results and turn off auto complete on search input

### CODE REVIEW

**Review by MarkusKuhn — APPROVED** [review]
Looks good on my side

File: offerzen/global/employers/employers@4.1.js [hunk 1]
```diff
@@ -388,7 +389,11 @@ window.$loaded(function (window, document, $, undefined) {
         url: '/search/skills?term=' + inputValue + '',
         contentType: 'application/json',
         success: function (skills) {
-          skillList = skills
+          skillList = skills.sort(function(a, b){
+            if(a.text < b.text) { return -1; }
+            if(a.text > b.text) { return 1; }
+            return 0;
+          })
```
Comment (jethroflanagan) [inline offerzen/global/employers/employers@4.1.js:396]:
```suggestion
          skillList = skills.sort(function(a, b){
            return a.text.localeCompare(b.text)
          })
```

**Review by jethroflanagan — APPROVED** [review]
You can simplify the sort. Your original version is also fine.

File: offerzen/global/employers/employers@4.1.js (hunk 1 above)
Comment (michael616kriel) [reply offerzen/global/employers/employers@4.1.js:396]: Cool this looks a lot better so ill commit your suggestion @jethroflanagan

**Pull request merged**